Release v0.1.0

WinterCore opened the pull request:
Changelog:
- Added navbar with CSS styling GH-1
- Added article section with CSS styling GH-2
- Added aside & footer sections with CSS styling GH-3
- Added `html-validator-cli` and `eslint`
- Added an `npm run lint` command to run all the linters (for HTML, CSS and JS)

santiago-rodrig commented:
## Pull Request under review by @santiago-rodrig (TSE)

santiago-rodrig commented on assets/css/style.css:
```diff
@@ -1,3 +1,257 @@
+  height: 100%;
+}
+
+nav ul li + li::before {
+  content: "";
+  display: block;
+  position: absolute;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 1px;
+  background: #4c4c4c;
+  box-shadow: 0 0 2px rgba(255, 255, 255, 0.8);
+}
```
Nice!

santiago-rodrig commented on assets/css/style.css:
```diff
@@ -1,3 +1,257 @@
+  background: linear-gradient(to bottom, #888 0%, #707070 49%, #5e5e5e 50%, #717571 100%);
+}
+
+nav ul .nav-logo a { background-position-y: 4px; }
+
+nav ul .nav-store a { background-position-y: -26px; }
+nav ul .nav-mac a { background-position-y: -56px; }
+nav ul .nav-ipod a { background-position-y: -86px; }
+nav ul .nav-iphone a { background-position-y: -116px; }
+nav ul .nav-ipad a { background-position-y: -146px; }
+nav ul .nav-itunes a { background-position-y: -176px; }
+nav ul .nav-support a { background-position-y: -206px; }
```
That's being a details person.

santiago-rodrig commented on assets/css/style.css:
```diff
@@ -1,3 +1,257 @@
+  height: 100%;
+}
+
+nav ul li:hover:not(:last-child),
+nav ul li:active:not(:last-child) {
+  background: -moz-linear-gradient(top, #5c5c5c 0%, #434343 49%, #383838 50%, #525252 100%);
+  background: -webkit-linear-gradient(top, #5c5c5c 0%, #434343 49%, #383838 50%, #525252 100%);
+  background: linear-gradient(to bottom, #5c5c5c 0%, #434343 49%, #383838 50%, #525252 100%);
+}
```
I recognize you have skills for the front-end

santiago-rodrig commented on assets/css/style.css:
```diff
@@ -1,3 +1,257 @@
+nav ul .nav-search-input {
+  padding: 0 13px;
+  width: 132px;
+  max-width: 132px;
```
```suggestion
```

santiago-rodrig commented on assets/css/style.css:
```diff
@@ -1,3 +1,257 @@
+  padding: 0 13px;
+  width: 132px;
+  max-width: 132px;
+  transition: all 0.3s ease-in-out;
```
Nice detail from your part

santiago-rodrig commented on assets/css/style.css:
```diff
@@ -1,3 +1,257 @@
+  transition: all 0.3s ease-in-out;
+}
+
+nav ul .nav-search-input .search-outer { position: relative; }
+
+nav ul .nav-search-input .search-outer::before {
+  content: "";
+  width: 14px;
+  height: 14px;
+  top: 3px;
+  left: 5px;
+  display: block;
+  position: absolute;
+  background: url(data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIGJhc2VQcm9maWxlPSJmdWxsIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB4bWxuczpldj0iaHR0cDovL3d3dy53My5vcmcvMjAwMS94bWwtZXZlbnRzIj4KPGcgZmlsbC1vcGFjaXR5PSIuNSI+Cgk8cG9seWdvbiBwb2ludHM9IjkuMjA3LDcuMTI2IDcuNzkzLDguNTQxIDExLjc5MywxMi41NDEgMTMuMjA3LDExLjEyNiIgLz4KCTxwYXRoIGQ ...
+  background-repeat: no-repeat;
+}
```
Great! :clap:

santiago-rodrig commented on assets/css/style.css:
```diff
@@ -1,3 +1,257 @@
+
+nav ul .nav-search-input.expanded {
+  width: 185px;
+  max-width: 185px;
```
```suggestion
```

WinterCore commented:
@santiago-rodrig Thank you for your review.
> Your project is great! But you shouldn't change the linters for others you personally find better. As a part of my duty, I must require you to change the linter configurations back to the ones suggested by Microverse, here you can find the files.

The `.eslintrc` and `stylelint.config.js` are identical to ones from the Microverse [linters config repo](https://github.com/microverseinc/linters-config)

The only change I did is that I added this to the `.stickler.yml` file
```yml
files:
  ignore:
      - 'stylelint.config.js'
```
And I'm pretty sure this isn't considered a change. I added it because eslint was trying to validate the `stylelint.config.js` file when it shouldn't be.

Thanks.

santiago-rodrig commented:
> @santiago-rodrig Thank you for your review.
> 
> > Your project is great! But you shouldn't change the linters for others you personally find better. As a part of my duty, I must require you to change the linter configurations back to the ones suggested by Microverse, here you can find the files.
> 
> The `.eslintrc` and `stylelint.config.js` are identical to the ones in the [Microverse linters config repo](https://github.com/microverseinc/linters-config)
> 
> The only change I did is that I added this to the `.stickler.yml` file
> 
> ```yaml
> files:
>   ignore:
>       - 'stylelint.config.js'
> ```
> 
> And I'm pretty sure this isn't considered a change. I added it because eslint was trying to lint the `stylelint.config.js` file when it shouldn't be.
> 
> Thanks.

Oh! Good move :+1:, I think it is now ready for approval, but that's not my duty now, another reviewer should take care of your project after you submit another CRR.

WinterCore commented:
I already submitted the pull request again for code review.
@santiago-rodrig Thank you so much for your time.
